J/jemalloc/build_tarballs.jl

```diff
@@ -17,6 +17,8 @@ autoconf
 
 if [[ "${target}" == *-freebsd* ]]; then
     ./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} --disable-initial-exec-tls --with-jemalloc-prefix
+elif [[ "${target}" == aarch64-apple-darwin* ]]; then
+    ./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} --disable-initial-exec-tls --with-lg-page=14
 else
     ./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} --disable-initial-exec-tls
 fi
```

Comment:
This is getting unmaintainable 
```suggestion
FLAGS=(--disable-initial-exec-tls)
if [[ "${target}" == *-freebsd* ]]; then
     FLAGS+=(--with-jemalloc-prefix)
elif [[ "${target}" == aarch64-apple-darwin* ]]; then
     FLAGS+=(--with-lg-page=14)
fi
./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} "${FLAGS[@]}"
```
And speaking of maintainability, leaving comments to explain what problem you're addressing and how the extra flags help solve it would make it easier to future (and current, I have little clue of what `--with-lg-page=14` does) readers what's going on.